src/collections/news/2023/2023-08-16-kubernetes-1.28-improves-open-source-cloud-native-compute-and-networking/index.mdx

```diff
@@ -0,0 +1,38 @@
+---
+title: " Kubernetes 1.28 improves open-source cloud-native compute and networking"
+subtitle: ""
+date: 2023-08-16 23:03:10 +0000
+category: Coverage
+author: SDxCentral
+thumbnail: ./kubernetes-1.28-improves-open-source-cloud-native-compute-and-networking.webp
+darkthumbnail: ./kubernetes-1.28-improves-open-source-cloud-native-compute-and-networking.webp
+eurl: "https://www.sdxcentral.com/articles/news/kubernetes-1-28-improves-open-source-cloud-native-compute-and-networking/2023/08/"
+published: true
+type: News
+---
+
+import { NewsWrapper } from "../../News.style.js";
+import { Link } from "gatsby";
+
+<NewsWrapper>
+
+The open-source Kubernetes cloud-native platform is out with its second major update of 2023, introducing a long list of enhancements for operators in the new 1.28 release.
+
+Kubernetes is an open-source project, originally started by Google and now developed under the Linux Foundation’s Cloud Native Computing Foundation (CNCF), benefiting from the contributions of more than 900 companies. Among the updates in Kubernetes 1.28 are enhancements designed to help with resiliency, including the ability to recover from non-graceful node shutdowns. There are also a series of networking-related enhancements that will enable better security and performance.
+
+As is the case with every Kubernetes release, there is a theme and a code name; for Kubernetes 1.28 the name is “Planternetes.” “So in the northern hemisphere, it’s summer right now and I think the symbolism of the garden and our community goes hand in hand,” Grace Nguyen, release lead for Kubernetes 1.28, told SDxCentral. “Each of us has a very important role in the ecosystem and together we built this big, open-source project that has a lot of impact.”
+
```

Comment:
@Ghat0tkach You missed adding the most essential section which mentions Meshery, layer, and @leecalcote. 

![image](https://github.com/layer5io/layer5/assets/65453826/bb410528-324b-49b4-aa61-e6fce1ba82b3)

Reply:
@Chadha93 Really apologies , Shall I add link to meshery and member profile of Lee as well?

Reply:
Yes.

Reply:
on my way